DNN Platform/Library/Entities/Urls/RewriteController.cs

```diff
@@ -1416,6 +1420,11 @@ internal static void GetUrlWithQuerystring(HttpRequest request, Uri requestUri,
             return result.DoRewrite;
         }
 
+        public static string GetUrlWithLanguageCode(string absoluteUri, string cultureCode)
```

Comment:
Can we use the [InternalsVisibleTo] attribute instead of making this public?

@dnnsoftware/approvers

Reply:
@dnnsoftware/approvers ☝️

Reply:
👍 sounds like the better approach